### DISCUSSION TITLE
KubeCommunity Day Paris 7 Mars 2023 🐳

### DISCUSSION THREAD

**Discussion by olivMertens** [lyonjs/shortvid.io#157] [Conference / Meetup Communication]
Hello There,
If there a possibility to have a template for this event ?
how can we help you or share assets ?
Thanks!

**Comment by mickaelalvs** [comment]
Hey ! Do you need only one video template? Something like what you put on twitter ? 

![image](https://user-images.githubusercontent.com/60877626/222679522-f5c29275-d8a9-4c81-b9c7-45a3f0880808.png)

**Reply by olivMertens** [reply]
yeah something with a fav icon photo some animations video and maybe music ?

**Reply by mickaelalvs** [reply]
Can you give us the different assets so that we can try to do a first test template ? 😊
